workspace/src/contract.rs

```diff
@@ -387,6 +389,47 @@ impl<U: UserFunctions> EvmAccount<U> {
         CallStorageWithdraw(self.near_call(&Call::StorageWithdraw).args_json(args))
     }
 
+    pub fn factory_update(&self, bytes: Vec<u8>) -> CallFactoryUpdate<'_> {
+        CallFactoryUpdate(
+            self.near_call(&OwnerCall::FactoryUpdate)
+                .args_borsh(Raw(bytes)),
+        )
+    }
+
+    pub fn factory_set_wnear_address(
+        &self,
+        address: impl Into<Address>,
```

Comment:
The address type from workspace-types definitely has the wrong declaration. Do not believe, that the initial implementation is correct. It isn't. I propose investigating `engine-types`. And use engine-compatible types.

Reply:
Can you explain why it's wrong? I thought the `H160` that this `Address` aliases to is correct.